Package.swift

```diff
@@ -0,0 +1,186 @@
+// swift-tools-version: 6.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "SubstrateSdk",
+    platforms: [.iOS(.v14)],
+    products: [
+        .library(
+            name: "SubstrateSdk",
+            targets: ["SubstrateSdk"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/novasamatech/Crypto-iOS", exact: "0.1.1"),
+        .package(url: "https://github.com/novasamatech/Operation-iOS", exact: "2.1.1"),
+        .package(url: "https://github.com/ashleymills/Reachability.swift", exact: "5.2.4"),
+        .package(url: "https://github.com/novasamatech/Starscream.git", exact: "4.0.13"),
+        .package(url: "https://github.com/bitmark-inc/tweetnacl-swiftwrap", exact: "1.1.0"),
+        .package(url: "https://github.com/attaswift/BigInt", exact: "5.5.1"),
+        .package(url: "https://github.com/daisuke-t-jp/xxHash-Swift", exact: "1.1.1"),
+        .package(url: "https://github.com/novasamatech/keccak.c", branch: "master"),
```

Comment:
Can we make a release of the `keccak.c`? I think it is safer to reference by release tag